cumulusci/tests/util.py

```diff
@@ -60,6 +61,14 @@ def __init__(
 
 class DummyOrgConfig(OrgConfig):
     def __init__(self, config=None, name=None, keychain=None, global_org=False):
+        if config is None:
+            config = {
+                "instance_url": "https://orgname.salesforce.com",
+                "access_token": "pytest_sf_orgconnect_abc123",
+                "id": "ORGID/ORGID",
```

Comment:
This should really in this form:

```suggestion
                "id": "https://test.salesforce.com/id/ORGID/USERID",
```

Reply:
135d4eb